### PR TITLE
Fix content localization route

### LINKED ISSUE #5129: How to preserve query string in language switcher
I have url a url like this `/en/media-center/sector?alias=sector-education-and-scholarships`.

Unfortunately `switch_culture_url` produces `https://localhost:44335/RedirectToLocalizedContent?targetculture=ar-EG&contentItemUrl=%2Fen%2Fmedia-center%2Fsector`, which eliminates the query string values. 

Is there any way to preserve the query string in the redirection?

### PR DESCRIPTION
**Pull request by hishamco**
While I'm trying to fix #5129, I got two issues, the first one is `OrchardCore.Modules.ContentLocalization` route is broken

This is simply the fix

### CODE REVIEW

**Comment by hishamco** [comment]
Seems the other issue fixed in https://github.com/OrchardCMS/OrchardCore/commit/cc677fa30d74c5f91af88571348d03a86058aa31

File: src/OrchardCore.Modules/OrchardCore.ContentLocalization/Startup.cs [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 using System.Globalization;
 using System.Linq;
-using Fluid;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Routing;
 using Microsoft.Extensions.DependencyInjection;
-using Microsoft.Extensions.DependencyInjection.Extensions;
+using Microsoft.Extensions.Options;
+using Fluid;
```
Comment (jtkech) [inline src/OrchardCore.Modules/OrchardCore.ContentLocalization/Startup.cs]: Sort usings, Fluid after System.Linq

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.ContentLocalization/Startup.cs]: Nope, System then Microsoft :)

Comment (jtkech) [reply src/OrchardCore.Modules/OrchardCore.ContentLocalization/Startup.cs]: I meant `using Fluid` just after `System.Linq`, not under `using Microsoft.Extensions.Options` ;)

But not so important, i think it's better to follow these rules but it should not block a PR.

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.ContentLocalization/Startup.cs]: AFAIK we sort usings as the following:

using System.*
using Microsoft.*

the remain usings

Comment (jtkech) [reply src/OrchardCore.Modules/OrchardCore.ContentLocalization/Startup.cs]: Nevermind but i'm using the `.editorconfig` of our repo and that (for usings) only contains

    # Sort using and Import directives with System.* appearing first
    dotnet_sort_system_directives_first                        = true

So when i use the right click `Remove and Sort Usings` command it puts Liquid upper.

Comment (hishamco) [reply src/OrchardCore.Modules/OrchardCore.ContentLocalization/Startup.cs]: Yep, that's why I need to sort them myself ;)

**Pull request merged**